api/v1_tracks.go

```diff
@@ -38,6 +38,16 @@ func (app *ApiServer) v1Tracks(c *fiber.Ctx) error {
 		ids = append(ids, newIds...)
 	}
 
+	// Add ISRC ID mappings
```

Comment:
two tracks can have the same ISRC, I think we should pick the one with the first `created_at`

Reply:
yeah. I think this probably "good enough"